### DISCUSSION TITLE
nghĩ phụ chủ đề khổ 3 koi

### DISCUSSION THREAD

**Discussion by 10A2DucAn01** [phhnhan-10A02-stt21/lop-10A02-Nhom8#24] [Q&A]
Miễn là em cố gắng
Không bao giờ đầu hàng
Bao khó khăn thử thách
Rồi cũng đến vinh quang

**Comment by NKLinhNgan-10A02-stt19** [comment]
lời động viên đến nv "em" ::))))

**Reply by 10A2DucAn01** [reply]
như động viên chính bản thân mình vậy^^

**Reply by LBUPhuong** [reply]
lấy ln làm gương nè

**Comment by 10A2DucAn01** [comment]
ai có câu chiện gì hay liên quan tới chủ đề khổ này quăng lên đây lun nka

**Reply by LBUPhuong** [reply]
để tìm thử đã ông đan

**Comment by LBUPhuong** [comment]
Tiêu đề là Xuất phát điểm của thành công

**Reply by 10A2DucAn01** [reply]
cụ thể hơn nữa đi

**Reply by LBUPhuong** [reply]
tiêu đề thì simple th m ^^

**Reply by 10A2DucAn01** [reply]
z còn phần cãm nhận
m có cãm nghĩ gì khổ này khum

**Reply by NKLinhNgan-10A02-stt19** [reply]
hmmm nêu ý nghĩa r động diên đồ đi

**Comment by LBUPhuong** [comment]
ủa còn đoạn 1 2 là đứa nào đặt tiêu đề chưa  ?

**Reply by NKLinhNgan-10A02-stt19** [reply]
chưa á m, t des mà::)))))

**Reply by 10A2DucAn01** [reply]
m đặt thử đi

**Reply by NKLinhNgan-10A02-stt19** [reply]
đou đoạn 1 2 đou

**Reply by 10A2DucAn01** [reply]
issues ák

**Reply by DoMinhQuanC2** [reply]
ròi ròi từ từ t đang xem
